README.md

```diff
@@ -31,7 +31,7 @@ logger.log(obj);
 * **token** 
     Mandatory. Your API logging token. Look it up in the Device Config tab in Logz.io
 * **type** - Log type. Help classify logs into different classifications
-* **protocol** - 'http' or 'https'. Default: http
+* **protocol** - 'http', 'https' or 'udp'. Default: http
```

Comment:
I think when we introduce udp, there are different sets of params. For example: bufferSize and numberOfRetries isn't not relevant. Maybe we should make it clear in the documentation which params are for http and which for UDP.